### DISCUSSION TITLE
How to differentiate between actual pinching by trackpad and ctrl+ scroll?

### DISCUSSION THREAD

**Discussion by myhoai** [pmndrs/use-gesture#506] [Q&A] [answered]
I try to use use-gesture/vanilla to zoom an image by pinching using the trackpad only (prevent ctrl + scroll to zoom).  Therefore I use PinchGesture.

```
new PinchGesture(
            imageContainer,
            pinchState => {
                console.log("pinch");
            },
            {
                target: image,
                scaleBounds: { min: 0.03, max: 1.25 },
            },
        );
```
When I zoom by the trackpad the Ctrl modifier key flag is set to true so I don't know how to detect pinch event using the trackpad.  I need some help here.

**Comment by dbismut** [comment, marked as answer]
The lib is not responsible for setting the control key to true when pinching your trackpad, so unfortunately there's no way to do what you want (and it's not because of this lib).
